rspace/src/main/scala/coop/rchain/rspace/history/Branch.scala

```diff
@@ -9,6 +9,8 @@ object Branch {
 
   val master: Branch = Branch("master")
 
+  val replay: Branch = Branch("replay")
```

Comment:
Constants should be in upper camel case by convention. Let's stick to that
https://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods

Reply:
Oh there's a convention here!  FANTASTIC.